### DISCUSSION TITLE
Is There a minimized version of the core?

### DISCUSSION THREAD

**Discussion by hazho** [GrapesJS/grapesjs#5824] [Q&A]
It would be really great to have the core of GJS without any components, except for the wrapper, if already there is a reale (of an LTS version) kindly mention that in the documentations

**Comment by hazho** [comment]
also without CodeMIRROR and other none-basic features

**Comment by artf** [comment]
[Work in progress](https://github.com/GrapesJS/grapesjs/discussions/5754)
